### PR TITLE
To Verfiy that list templates by domain admin is not listing the templates of other domains...

### PR DESCRIPTION
**Pull request by manasaveloori**
results:

test_listtemplate (integration.component.test.TestlistTemplatesDomainAdmin) ... === TestName: test_listtemplate | Status : SUCCESS ===
ok

----------------------------------------------------------------------
Ran 1 test in 307.996s

OK

### CODE REVIEW

File: test/integration/component/test_escalation_listTemplateDomainAdmin.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            if isinstance(listTemplateResponse, list):
+                break
+            elif timeout == 0:
+                raise Exception("list template failed")
+
+            time.sleep(10)
+            timeout = timeout - 1
+
+            self.assertEqual(
+                isinstance(listTemplateResponse, list),
```
Comment (gauravaradhye) [inline test/integration/component/test_escalation_listTemplateDomainAdmin.py]: isinstance and len() check for list can be combined by using validateList method from common.py. Use only single method.

File: test/integration/component/test_escalation_listTemplateDomainAdmin.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+            templatefilter=self.testdata["templatefilter"]
+
+        )
+
+        self.assertEqual(
+            listtemplate,
+            None,
+            "Check templates are not listed"
+        )
+        return
```
Comment (gauravaradhye) [inline test/integration/component/test_escalation_listTemplateDomainAdmin.py]: All other changes looks good.

**Comment by DaanHoogland** [comment]
@gauravaradhye will you do the honors?

**Comment by bhaisaab** [comment]
LGTM

**Comment by bhaisaab** [comment]
Patch fails, file already exists? Please rebase and push again, or close PR if this is already merged on master in some other PR

**Pull request closed without merging**